app/controllers/katello/concerns/api/v2/dynamic_params/repositories.rb

```diff
@@ -0,0 +1,20 @@
+module Katello
+  module Concerns
+    module Api::V2
+      module DynamicParams
+        module Repositories
+          extend ::Apipie::DSL::Concern
+
+          lazy_update_api(:import_uploads) do
```

Comment:
`lazy_update_api` doesn't exist in `apipie-rails` yet, the PR is open: https://github.com/Apipie/apipie-rails/pull/955